src/main/java/org/sqlite/core/DB.java

```diff
@@ -695,6 +695,15 @@ public final synchronized int finalize(CoreStatement stmt) throws SQLException {
      */
     public abstract int restore(String dbName, String sourceFileName, ProgressObserver observer) throws SQLException;
 
+    /**
```

Comment:
Is this new?

Reply:
Yes this is new [DB.java](https://github.com/xerial/sqlite-jdbc/blob/3.32.3.2/src/main/java/org/sqlite/core/DB.java) line 696

Reply:
Nope, I was looking at the snapshot for 3.32.3.3, the other method in NativeDB.java is also related to the newest snapshot